pkgs/applications/emulators/duckstation/default.nix

```diff
@@ -20,18 +20,17 @@
 , vulkan-loader
 , wayland
 , wrapQtAppsHook
-, enableWayland ? true
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+llvmPackages_16.stdenv.mkDerivation (finalAttrs: {
```

Comment:
It does no work with typical stdenv? Why?

Reply:
To follow upstream more closely (the AppImages are built with LLVM).

Reply:
Is this somehow mandatory? "Duckstation does not accept bug reports when built with GCC" or something like that?

Reply:
It actually doesn't accept bug reports outside the official Flatpak or AppImage period.
So no, it's not mandatory.
Looking at the nixpkgs manual, I couldn't find anywhere whether it's preferred to follow upstream for these kinds of things.

Reply:
Well, I have few knowledge about the innards of Nixpkgs, but, intuitively, using `llvm16Packages.stdenv` makes the expression less generalist, by pinning a toolchain (and a specific version of that toolchain). It reduces the options and offers no sensible advantage.
I see no reason to do this.